### ISSUE TITLE
WARNING Log - Websocket closed : WSMsgType.CLOSE

### ISSUE THREAD

**Issue by calanthe23** [bodyscape/cielo_home#10]
I noticed this WARNING message has been occurring in my logs.  There do not seem to be any issues with the integration for me, but I wanted to report it just in case it needed to be looked into.  Thanks!

```
This error originated from a custom integration.

Logger: custom_components.cielo_home.cielohome
Source: custom_components/cielo_home/cielohome.py:204
Integration: Cielo Home (documentation, issues)
First occurred: May 6, 2023 at 8:09:46 AM (87 occurrences)
Last logged: 12:19:10 PM

Websocket closed : WSMsgType.CLOSE
Websocket closed : WSMsgType.CLOSED
```

**Comment by calanthe23** [comment]
Potentially related, I have also been seeing this in the logs:
`Logger: root
Source: helpers/entity.py:724
First occurred: May 24, 2023 at 6:07:23 AM (175 occurrences)
Last logged: 6:04:29 AM

Uncaught thread exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.10/threading.py", line 1378, in run
    self.function(*self.args, **self.kwargs)
  File "/config/custom_components/cielo_home/cielohomedevice.py", line 756, in dispatch_state_updated
    listener.state_updated()
  File "/config/custom_components/cielo_home/entity.py", line 37, in state_updated
    self.schedule_update_ha_state(False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 724, in schedule_update_ha_state
    self.hass.loop.call_soon_threadsafe(self.async_write_ha_state)
AttributeError: 'NoneType' object has no attribute 'loop'
`

**Comment by bodyscape** [comment]
fix on this commit : a254e736775075c0636e1891674842a959c1b5a2
